src/header/Header.stories.tsx

```diff
@@ -0,0 +1,273 @@
+/**
+ * Copyright (c) 2018 Dell Inc., or its subsidiaries. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+import * as React from "react";
+import {storiesOf} from "@storybook/react";
+import {action} from "@storybook/addon-actions";
+import {Icon} from "../icon";
+import {Dropdown, DropdownMenu, DropdownItem} from "../forms/dropdown";
+import {Header, HeaderColor} from "./Header";
+import {NavLink} from "../layout/nav";
+// <NavLink>Link 1</NavLink>
+
+storiesOf("Header", module)
+    .add("Header Types", () => (
+        <React.Fragment>
+            <div>
+                <h4> Header Types</h4>
+                <Header className={HeaderColor.Header6} style={{marginTop: "24px"}}>
+                    <div className="branding">
+                        <NavLink>
+                            <Icon shape="vm-bug" />
+                            <span className="title">Project Clarity</span>
+                        </NavLink>
+                    </div>
+                    <div className="header-nav">
+                        <NavLink>
+                            <span className="nav-text">Dashboard</span>
+                        </NavLink>
+                        <NavLink>
+                            <span className="nav-text">Interactive Analytics</span>
+                        </NavLink>
+                    </div>
+                    <div className="header-actions">
+                        <NavLink>
+                            <Icon shape={"cog"} />
+                        </NavLink>
+                    </div>
+                </Header>
+
+                <Header className={HeaderColor.Header6} style={{marginTop: "24px"}}>
+                    <div className="branding">
+                        <NavLink>
+                            <Icon shape="vm-bug" />
+                            <span className="title">Project Clarity</span>
+                        </NavLink>
+                    </div>
+                    <form className="search">
+                        <label>
+                            <input id="search_input" type="text" placeholder="Search for keywords#" />
+                        </label>
+                    </form>
+                    <div className="header-actions">
+                        <NavLink>
+                            <Icon shape="cog" />
```

Comment:
We will need to handle a click event here somehow.  Can you enhance the stories to have working click events?  For example, in our new UI we'll want to click a bell icon and trigger a GraphQL query for listing events.  How would that work here?

Reply:
@ben-schumacher  I have added handler for 'NavLink' Click event as part of Wizard PR. https://github.com/EMCECS/clarity-react/pull/31/files#diff-7ddafb8b9b54a855efffac95d7f02277R18

We can enhance this story once PR for Wizard gets merged or I will take care of it as part of Wizard PR.